pkg/util/fileutil/filesystem_test.go

```diff
@@ -64,7 +64,29 @@ func TestGetMountedFiles(t *testing.T) {
 				return dir
 			},
 			expectedErr: false,
-			expectedKey: "subdir/secret.txt",
+			want:        []string{"subdir/secret.txt"},
```

Comment:
on windows:

```
    --- FAIL: TestGetMountedFiles/target_path_dir/dir/file_found (0.00s)
        filesystem_test.go:106: GetMountedFiles() keys mismatch (-want +got):
              []string{
            -   "subdir/secret.txt",
            +   `subdir\secret.txt`,
              }
        tmpdir.go:39: error cleaning up tmp dir: remove C:\Users\tommy\AppData\Local\Temp\ut2093061619\subdir\secret.txt: The process cannot access the file because it is being used by another process.
```

Reply:
I think the `targetPath` func() for this test case needs to get the return values of os.Create and call `f.Close()`